scripts/metavar.py

```diff
@@ -1427,7 +1427,7 @@ def __init__(self, name, run_env, variables=None,
                 self[stdname] = variables[key]
             # end for
         elif variables is not None:
-            raise ParseInternalError('Illegal type for variables, {} in {}'.format(type(variables), self.name))
+            raise ParseInternalError('Illegal type for variables, {} in {}'.format(type_name(variables), self.name))
```

Comment:
Since we're in here, would it make sense to turn this into an `f"...` formatted string as well?

Reply:
changed!